src/bgw/job.c

```diff
@@ -14,7 +14,7 @@
 #include "telemetry/telemetry.h"
 
 const char *job_type_names[_MAX_JOB_TYPE] = {
```

Comment:
This doesn't seem to be referenced outside this file, is it supposed to be `static`?